packages/cli/src/daemon-config.ts

```diff
@@ -316,12 +322,24 @@ export class DaemonCeramicNodeConfig {
     return this._privateSeedUrl
   }
 
+  /**
+   * http fetch timeout
+   */
+  @jsonMember(Number, { name: 'http-fetch-timeout' })
+  httpFetchTimeout?: number
```

Comment:
i would like to use this in ./packages/common/src/utils/http-utils.ts but its not clear to me how to best do that - @stbrody would love thoughts on how to set up configuratble flags and feature flags more generally, is daemon config the right way to do it?

Reply:
@stephhuynh18 maybe you have an opinion?